mappings/net/minecraft/entity/ai/TimeRange.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4802 net/minecraft/entity/ai/TimeRange
```

Comment:
`IntRangeUtil`? This class is an odd one :P

Reply:
The other one has like tick time range, this one is like return a second time range

Reply:
I called it `IntRangeFactory`, but all 3 of the names could be improved. `SecondRange` maybe?

Reply:
Renamed it to `Durations`